DocAI Incubator Tools/best-practices/Pre_and_Post_HITL_Visualization.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #154.    def bbox_maker(boundingPoly):
</pre><p>use snake_case for variable names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>